tests/testthat/indention_square_brackets/square_brackets_line_break-out.R

```diff
@@ -5,53 +5,54 @@ ranges[
 
 fak[a, b]
 
-fac[
-  a,
-  b
-]
+fac[a, b]
 
 fac[
   a,
   b
 ]
 
-fac[
-  , `:`(a = b)
+fac[,
+  `:`(a = b)
 ]
 
-fac[
-  , `:`(a = b)
+fac[,
+  `:`(a = b)
 ]
 
 fac[, `:`(a = c)]
 
-fac[
-  , `:`(a = c)
+fac[,
+  `:`(a = c)
 ]
 
-x[a == 3 |
-  b == v, ]
+x[
+  a == 3 |
+    b == v,
+]
```

Comment:
We discussed this yesterday with @DavisVaughan but seeing it here I wonder if we should take another look...

Reply:
@hadley recommends fixing this: https://github.com/posit-dev/air/issues/232